presenters/keyword_test.go

```diff
@@ -0,0 +1,63 @@
+package presenters_test
+
+import (
+	"encoding/json"
+
+	_ "google-scraper/initializers"
```

Comment:
@junan Do we really need this one?
Because it looks cycle import as you've already imported in this https://github.com/junan/google-scraper/blob/develop/controllers/controllers_suite_test.go#L6 🤔

Reply:
That one is actually in the controller, and this one in the presenters. So I need it here.